src/main/java/com/amaze/filemanager/utils/Futils.java

```diff
@@ -680,90 +680,128 @@ public void onSelection(MaterialDialog materialDialog, View view, int i, CharSeq
         }
     }
 
+    @SuppressWarnings("ConstantConditions")
     public void deleteFiles(ArrayList<LayoutElement> layoutElements, final MainFragment mainFragment, List<Integer> positions, AppTheme appTheme) {
-        final MaterialDialog.Builder dialogBuilder = new MaterialDialog.Builder(mainFragment.getActivity());
-        dialogBuilder.title(mainFragment.getResources().getString(R.string.confirm));
-
         int counterDirectories = 0;
         int counterFiles = 0;
-        long longSizeTotal = 0;
+        long sizeTotal = 0;
         final ArrayList<BaseFile> itemsToDelete = new ArrayList<>();
-        StringBuilder listDirectories = new StringBuilder();
-        StringBuilder listFiles = new StringBuilder();
+        StringBuilder builderDirectories = new StringBuilder();
```

Comment:
There's no _need_ for it, but prepending `builder` might help understand how that variable is used.  I can remove that if it's a big deal.  Regardless, debating variable names can lead to [bikeshedding](https://en.wiktionary.org/wiki/bikeshedding).